script/openqa-clone-custom-git-refspec

```diff
@@ -78,14 +79,21 @@ if [[ -z "$branch" ]] || [[ -z "$repo_name" ]]; then
     repo_name="${repo_name:-"${label%:*}/${target_repo_part##*/}"}"
     branch="${branch:-"${label##*:}"}"
     repo="${repo:-"https://github.com/${repo_name}.git"}"
+
+    # Pick test(s) to run from the PR description
```

Comment:
Because you already separate into a block with a comment on top you could extract a function with a proper name so that not even the comment would be necessary anymore